app/models/purchase.py

```diff
@@ -45,23 +45,54 @@ class Order(db.Model):
     date = db.Column(db.DateTime)
     status = db.Column(db.Integer)
 
-    def __init__(self, cart_items):
-        self.date = datetime.now(pytz.timezone('US/Eastern'))
+    vendor_id = db.Column(db.Integer)
+    company_name = db.Column(db.String(64))
+
+    def __init__(self, date, vendor_id):
         self.status = Status.PENDING
+        self.date = date
```

Comment:
This I'm not super committed to keep, but I was thinking that when a cart is checked out, the individual orders placed (for each vendor in the "overall order") should all have the same exact time. So with this we can pass in the same date for all of the individual orders. What do you think?